source/reference/verification.txt

```diff
@@ -0,0 +1,123 @@
+
+====================
+Verify Data Transfer
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Following a sync, verify the successful transfer of your data before you switch
```

Comment:
I think we should make it clear that we mean once mongosync has fully committed.

Reply:
Fixed